config/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
 if(BUILD_DEB)
     install(FILES systemd/aktualizr-ubuntu.service
-        DESTINATION lib/systemd/system
+        DESTINATION /lib/systemd/system
```

Comment:
Why the leading slash? I thought there was a reason we didn't have it, and it isn't present for the other files.

Reply:
So, with a slash it's an absolute file on the target file system. Lintian complains about installing systemd services in `/usr/lib` instead of `/lib` which is what is documented in `systemd.unit(5)`: https://lintian.debian.org/tags/systemd-service-file-outside-lib.html

More precisely when running `make/ninja install`, `/xx` should install in `$DESTDIR/xx` and `xx` in `$DESTDIR/$PREFIX/xx`. `$DESTDIR` defaults to `/` and `$PREFIX` to `/usr/local`.

I know that we've removed absolute paths in cmake's installs, notably some starting with `/etc`, and this is going in the other direction.

Reply:
Fair enough. I'll trust you and the linter!